docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.2.0-RC1, with new optimizations, improved stability and IDE support
+author: Dmytro Petrashko
+authorImg: /images/petrashko.jpg
+date: 2017-05-31
+---
+
+Today, we are excited to release Dotty version 0.2.0-RC1. This release
+serves as a technology preview that demonstrates new language features
+and the compiler supporting them.
+
+This release is based on [previous milestone](/blog/2017/05/31/first-dotty-milestone-release.html).
+The highlights of this release are:
+ - substantial improvement of quality of generated code for pattern matching
+ - improvements in VS Code IDE stability
+ - improved compatibility with scalac 
+ - initial support for reproducible builds
+
+
+<!--more-->
+
+This is our second scheduled release according to our [6-week release schedule](http://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+# What’s in the 0.2.0-RC1 technology preview?
+The [previous technology preview](/blog/2017/05/31/first-dotty-milestone-release.html) has shipped new language planned for Scala 3:  
+[Intersection Types](http://dotty.epfl.ch/docs/reference/intersection-types.html),
+[Union Types](http://dotty.epfl.ch/docs/reference/union-types.html),
+[Trait Parameters](http://dotty.epfl.ch/docs/reference/trait-parameters.html),
+[Enumerations](http://dotty.epfl.ch/docs/reference/enums/enums.html),
+[Algebraic Data Types](http://dotty.epfl.ch/docs/reference/enums/adts.html),
+[By-Name Implicits](http://dotty.epfl.ch/docs/reference/implicit-by-name-parameters.html).
+
+This technology preview is geared towards improving stability and reliability. It includes:
+
+  - [Local optimizations upstreamed from Dotty Linker](https://github.com/lampepfl/dotty/pull/2513), [2647](https://github.com/lampepfl/dotty/pull/2647) by ([@OlivierBlanvillain](https://github.com/OlivierBlanvillain). See more details below.
+  - [Optimizing Pattern Matcher](https://github.com/lampepfl/dotty/pull/2829) by ([@odersky](https://github.com/odersky)
+  - [Idempotency checks](https://github.com/lampepfl/dotty/pull/2756) are the first step to reproducible builds
+  - [Faster Base class sets](https://github.com/lampepfl/dotty/pull/2676) by ([@odersky](https://github.com/odersky) and ([@darkdimius](https://twitter.com/darkdimius) 
+  - Huge number of fixes to IDE and Dotty Language Server covering:
```

Comment:
I think the most important IDE change that isn't mentioned in this list is that it now supports Windows: https://github.com/lampepfl/dotty/pull/2776